lib/engine/step/g_18_ms/bankrupt.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
this file seems like it's not needed

Reply:
See above - would result in an unhandled action without changes of the view code.